packages/test-commerce-sdk-react/app/components/_app-config/index.tsx

```diff
@@ -30,9 +30,10 @@ const AppConfig = (props: AppConfigProps): ReactElement => {
     return (
         <CommerceApiProvider
             siteId={siteId}
-            shortCode="8o7m175y"
-            clientId="c9c45bfd-0ed3-4aa2-9971-40f88962b836"
-            organizationId="f_ecom_zzrf_001"
+            // TODO: On Feb 08 restore zzrf_001 details
+            shortCode="sandbox-001"
+            clientId="06bb7d20-93fa-4707-ade6-2ecd858331bd"
+            organizationId="f_ecom_bjnl_dev"
```

Comment:
Today we expected to have the endpoint available in `zzrf_001`, but I tested now it's  still returning 404s.

Do we want to merge with the TODO or restore the `zzrf_001` config assuming the endpoint will be available at some point?